app/lib/system/error_reporting.rb

```diff
@@ -11,12 +11,9 @@ module ErrorReporting
       options
     end
 
-    def report_error(exception, rack_env: nil, logger: Rails.logger, **parameters)
-      options = OPTIONS.call(parameters, rack_env)
-
+    def report_error(exception, logger: Rails.logger, **parameters)
```

Comment:
Oops, this change will break https://github.com/3scale/porta/blob/c1fbcb787ff4b4d8f257a06a7a22705d0ec08563/app/lib/three_scale/message_bus_config.rb#L28

Reply:
Right, so we just need to remove that parameter from that line too :smile: 
![image](https://user-images.githubusercontent.com/11318903/69839768-465b4f80-1259-11ea-9c45-
761d588af2e6.png)

Reply:
What about [OPTIONS](https://github.com/3scale/porta/blob/a9bd838dbcd11d1594c2dbfad6330745547067ab/app/lib/system/error_reporting.rb#L7-L11)? Where is it used?

Reply:
@josemigallas it is not used at all anymore. u can remove it :smile: